### PR TITLE
Complete-ish

### PR DESCRIPTION
**Pull request by austie702**
Focusing on securities this week as it pertains to my project's deadline. 

I copied the code from Abby Tiffany for the CA code so that I could experiment with the program and test results. Managed to make lots of great versions of the pattern using her code and was able to reverse-engineer the setup very cleanly. (Abby if you read this, great code! Very succinct and readable). 

Securities are going very well! I have more to document and study, but this is a helpful file showing options and instructions on where to work from to handle threat assessments.

### CODE REVIEW

File: security/security.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
+# Threat Model:
+
+### Step 01: Decompose the application
+* How does the app interact with external entities?
+* create use-cases to understand how the application is used, 
+* identifying entry points to see where a potential attacker could interact with the app
+* identifying assets (i.e. items/areas that an attacker would be interested in
```
Comment (JCamp-12) [inline security/security.md:8]: attack surface

File: security/security.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* identifying trust levels which represent the access rights that the app will grant to external entities
```
Comment (JCamp-12) [inline security/security.md:9]: this reads very cleanly, I liked the flow and how thorough this was.

File: security/security.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
```
Comment (JCamp-12) [inline security/security.md:10]: nice use of .md

**Review by JCamp-12 — COMMENTED** [review]
great job Austin, you copied from the right person.  That answer was great, my takeaway is that you get security really well and after getting over the initial hump adding rules is not that bad.  Not the worst takeaway to focus on security, like your approach last couple weeks bud.  Keep working hard!!

**Review by agallant — COMMENTED** [review]
:+1: agreed that this is a solid start to documenting the security/threat model of your project. I'll close this out for now as it definitely satisfies the sprint challenge, and if you need further feedback down the line you can push commits and reopen here if you want, or just add it to whatever project repo and share the link. Thanks!

**Pull request closed without merging**